languages/elixir/exercises/concept/conditionals/.docs/after.md

```diff
@@ -0,0 +1,19 @@
+You can use atoms whenever you have a set of constants you want to express. Using atoms opens a type-safe way of interacting with constant values. An atom is defined by its name, follwing the atom format:
```

Comment:
```suggestion
You can use atoms whenever you have a set of constants you want to express. Using atoms opens a type-safe way of interacting with constant values. An atom is defined by its name, following the atom format:
```

Reply:
The `hints.md` file has a nice link to a document on atoms. Could you also link to that document somewhere in this document?